runtime-light/stdlib/file/php-streams/php-streams.cpp

```diff
@@ -0,0 +1,30 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2024 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#include "runtime-light/stdlib/file/php-streams/php-streams.h"
+
+#include <cstring>
+
+#include "runtime-light/state/instance-state.h"
+
+namespace {
+
+constexpr std::string_view STDERR_NAME{"stderr"};
+constexpr std::string_view STDOUT_NAME{"stdout"};
+constexpr std::string_view STDIN_NAME{"stdin"};
+
+} // namespace
+
+uint64_t open_php_stream(const string &stream_name) noexcept {
+  if (strcmp(stream_name.c_str(), STDERR_NAME.data()) == 0) {
```

Comment:
You do have `std::string_view` here, why not just create another one and compare string views instead of raw pointers?

`if (std::string_view{stream_name.c_str(), stream_name.size()} == STDERR_NAME)`